sql/core/src/test/scala/org/apache/spark/sql/connector/MetadataColumnSuite.scala

```diff
@@ -17,12 +17,27 @@
 
 package org.apache.spark.sql.connector
 
+import java.io.{File, FilenameFilter}
+
 import org.apache.spark.sql.{AnalysisException, Row}
+import org.apache.spark.sql.connector.catalog.{Identifier, InMemoryCatalog, InMemoryTable, MetadataColumn, Table}
+import org.apache.spark.sql.connector.expressions.Transform
 import org.apache.spark.sql.functions.struct
+import org.apache.spark.sql.types.{DataType, IntegerType, StringType, StructField, StructType}
 
 class MetadataColumnSuite extends DatasourceV2SQLBase {
   import testImplicits._
 
+  override protected def beforeEach(): Unit = {
```

Comment:
It has to be a `beforeEach` as `DatasourceV2SQLBase` resets the spark settings after every test